src/components/About/Support/index.js

```diff
@@ -185,8 +185,7 @@ const SignUpFooter = styled.div`
 
   @media (min-width: 1400px) {
     position: relative;
-    top: 6.25rem;
-    padding: 0 0 9.375rem 0;
+    padding: 3.125rem 0 3.125rem 0;
```

Comment:
This is a hard coded solution and doesn't work on various smaller screen sizes. I think you can either add more media queries tracking those breakpoints or find another way around.

Reply:
I don't think it's hard-coded it was actually already prewritten [here](https://github.com/fossasia/susi.ai/blob/68f770103ca50c4acfe5b3a728218194f2886e78/src/components/About/Support/index.js#L184)(for all sizes) & I just rectified it in min width(for sizes greater than 1400px) since the padding-down & top value was mistakenly more so was overlapping with links,so basically no problem for smaller sizes.(i have checked it too)